code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -349,6 +349,13 @@ GLOBAL_LIST_EMPTY(lifts)
 
 				playsound(collided_wall, 'sound/effects/meteorimpact.ogg', 100, TRUE)
 
+			if(ismineralturf(dest_turf))
+				var/turf/closed/mineral/M = dest_turf
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names